design.md

```diff
@@ -0,0 +1,43 @@
+# Epiverse search
+
+## User persona & user story
+
+This work is mostly targeted at the following persona, with the following stories:
+
+- As a PEI tool implementer, I want to find out if there is an R package to do this specific epidemiological analysis task I have in mind. This allows me to quickly find the right tool for the job.
+- As a PEI tool creator, I want to find out if there is an R package to do this specific epidemiological analysis task I have in mind. This prevents me from reinventing the wheel and increasing the ecosystem fragmentation.
+
+These user stories have been collected at:
+
+- the '100 days and 100 lines of code' workshop (December 2022)
+- the 'Solution for Collaboratory' launch workshop (January 2023)
+
+## Requirements
```

Comment:
Do we have any requirements on the technology stack to create the search? Or are these open to being scoped out during the process itself?

My main thoughts for scope are for example whether they should be based on open source software, free(mium) services, etc. If the requirement is that it should be reproducible in other environments without major hurdles, that would also already help (e.g., this would scope away proprietary tools or licenses needed).